src/inmanta/config.py

```diff
@@ -262,6 +262,14 @@ def is_list(value: str | list[str]) -> list[str]:
     return [] if value == "" else [x.strip() for x in value.split(",")]
 
 
+def is_lower_bounded_int(lower_bound: int, value: str | int) -> int:
+    """lower-bounded int"""
+    to_int = int(value)
+    if to_int < lower_bound:
+        raise ValueError(f"Value can not be lower than {lower_bound}")
+    return to_int
```

Comment:
to make the docs prettier

```suggestion
def is_lower_bounded_int(lower_bound: int) -> Callable[[str | int],int]:
    """lower-bounded int factory"""
    def inner(value: str | int):
         to_int = int(value)
         if to_int < lower_bound:
              raise ValueError(f"Value can not be lower than {lower_bound}")
         return to_int
    inner.__doc__ = f"int < {lower_bound}"
    return inner 
```